apps/ejabberd/src/mod_muc_light_utils.erl

```diff
@@ -162,18 +162,27 @@ filter_out_prevented(FromUS, {RoomU, MUCServer} = RoomUS, AffUsers) ->
 %% ---------------- Filter for blocking ----------------
 
 -spec filter_out_loop(FromUS :: ejabberd:simple_bare_jid(),
-                      BlockingQuery :: [{blocking_who(), ejabberd:simple_bare_jid()}],
+                      MUCServer :: ejabberd:lserver(),
+                      BlockingQuery :: [{blocking_what(), ejabberd:simple_bare_jid()}],
                       RoomsPerUser :: rooms_per_user(),
                       AffUsers :: aff_users()) -> aff_users().
-filter_out_loop(FromUS, BlockingQuery, RoomsPerUser, [{UserUS, _} = AffUser | RAffUsers]) ->
-    case (BlockingQuery == undefined orelse UserUS =:= FromUS
-          orelse ?BACKEND:get_blocking(UserUS, BlockingQuery) == allow)
-         andalso
-         (RoomsPerUser == infinity orelse length(?BACKEND:get_user_rooms(UserUS)) < RoomsPerUser) of
-        true -> [AffUser | filter_out_loop(FromUS, BlockingQuery, RoomsPerUser, RAffUsers)];
-        false -> filter_out_loop(FromUS, BlockingQuery, RoomsPerUser, RAffUsers)
+filter_out_loop(
```

Comment:
How about following line split:

```erlang
filter_out_loop(FromUS, MUCServer, BlockingQuery,
                         RoomsPerUser, [{UserUS, _} = AffUser | RAffUsers])
```

Look the comment "source code" as Github renders it incorrectly.